src/components/Header/Header.js

```diff
@@ -9,10 +9,10 @@ const Header = () => {
       <h1 className='glow'>W o r m h o l e</h1>
       <nav>
         <Link exact to='/'>
-          <button className='random-location'>Randomize Location</button>
+          <button className='nav-btn'>Randomize Location</button>
         </Link>
         <Link exact to='/'>
-          <button className='go-home'>Return Home</button>
+          <button className='nav-btn'>Return Home</button>
```

Comment:
i will adjust these styling names to fit the cypress tests when i pull down; I will maintain your styles.